src/CribblyBackend/Controllers/TeamController.cs

```diff
@@ -97,5 +99,33 @@ public async Task<IActionResult> GetByTeamId(int id)
             _logger.Information("Request for games from team {id} returned no results", id);
             return NotFound();
         }
+
+        /// <summary>
+        /// Delete permanently removes the team with the specified id.
+        /// </summary>
+        /// <param name="id">The Team object that will be deleted from the database</param>
+        /// <returns></returns>
+        [HttpDelete("{id}")]
+        public async Task<IActionResult> Delete([FromBody] Team team)
```

Comment:
Let's choose _either_ a URL param or something in the body. I think for deletes, it's more common to put the ID in the URL.

Reply:
Fixed?